### PR TITLE
Adding an upgrade test lane

### PR DESCRIPTION
**Pull request by omeryahud**
Signed-off-by: Omer Yahud <oyahud@redhat.com>

### CODE REVIEW

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml [hunk 1]
```diff
@@ -35,6 +35,26 @@ tests:
...
     workflow: ipi-aws
+- as: e2e-upgrade-functests
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-upgrade-functests
+      cli: latest
+      commands: |
+        make deploy-kubevirt # Should deploy Kubevirt and CDI and wait for their CRs to report 'Available'
+        make deploy-kubevirt-ssp-operator # Should deploy kubevirt-ssp-operator (https://github.com/kubevirt/kubevirt-ssp-operator) and wait for its 4 CRs to report 'Available'
```
Comment (akrejcir) [inline ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml:46]: These could be shell scripts in the `hack` directory, or `automation` directory, not make targets.

**Comment by akrejcir** [comment]
/hold
Holding until PR with upgrade tests is merged.
https://github.com/kubevirt/ssp-operator/pull/42

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml (hunk 1 above)
Comment (omeryahud) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-master.yaml:46]: Yea that's better, will change

**Comment by omeryahud** [comment]
> /hold
> Holding until PR with upgrade tests is merged.
> [kubevirt/ssp-operator#42](https://github.com/kubevirt/ssp-operator/pull/42)

We should actually merge it before, so the PR can run the lane (and ignore it on other PRs for now)

**Comment by akrejcir** [comment]
Or better, I will post a new PR with the scripts, and this PR will wait for the new one.

**Comment by akrejcir** [comment]
I have opened an updated version of this PR: #14010

/close

**Pull request closed without merging**